Failed tests (11084): TestAsyncSchemaChanger

The following test appears to have failed:

[#11084](https://circleci.com/gh/cockroachdb/cockroach/11084):

```
127.0.0.1:0  08:57:40.189831  1.015559ms  sending batch                 kv/txn_coord_sender.go:308
127.0.0.1:0  08:57:40.189847  13.192µs    ·meta descriptor lookup       kv/dist_sender.go:562
127.0.0.1:0  08:57:40.189872  966.831µs   ·sending RPC                  kv/dist_sender.go:422
127.0.0.1:0  08:57:40.189957  0           ··sending to 127.0.0.1:58382  rpc/send.go:154

panic: test timed out after 1m10s

goroutine 26941 [running]:
testing.startAlarm.func1()
	/usr/local/go/src/testing/testing.go:703 +0x132
created by time.goFunc
	/usr/local/go/src/time/sleep.go:129 +0x3a

goroutine 1 [chan receive]:
testing.RunTests(0x1769fc8, 0x1e03a80, 0x31, 0x31, 0x1563801)
	/usr/local/go/src/testing/testing.go:562 +0x8ad
testing.(*M).Run(0xc820963f08, 0xc8201ea820)
	/usr/local/go/src/testing/testing.go:494 +0x70
github.com/cockroachdb/cockroach/util/leaktest.TestMainWithLeakCheck(0xc820963f08)
	/go/src/github.com/cockroachdb/cockroach/util/leaktest/leaktest.go:37 +0x21
github.com/cockroachdb/cockroach/sql_test.TestMain(0xc820963f08)
	/go/src/github.com/cockroachdb/cockroach/sql/main_test.go:48 +0x21
main.main()
	github.com/cockroachdb/cockroach/sql/_test/_testmain.go:190 +0x113

goroutine 17 [syscall, 1 minutes, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1721 +0x1

goroutine 20 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x214dee0)
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:1029 +0x64
created by github.com/cockroachdb/cockroach/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:610 +0xbc

goroutine 25530 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:622 +0x3bf
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820a60c20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25512 [select]:
github.com/cockroachdb/cockroach/sql.(*LeaseManager).RefreshLeases.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/lease.go:624 +0xe0f
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820a036e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25555 [IO wait]:
net.runtime_pollWait(0x7f0cadb549d8, 0x72, 0xc82007e0b0)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820206bc0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820206bc0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820206b60, 0xc8204be000, 0x400, 0x400, 0x0, 0x7f0cadb44028, 0xc82007e0b0)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc820082490, 0xc8204be000, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func2(0xc8205b5140, 0x7f0cb039ef80, 0xc8200cb080, 0x7f0cb03a39b0, 0xc8202d7b00, 0xc82021e5c0, 0xc8206f2900)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:210 +0x5d
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:213 +0x90c

goroutine 25651 [select]:
github.com/cockroachdb/cockroach/storage.(*idAllocator).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/id_alloc.go:129 +0x93e
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc8202c9550)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25551 [select]:
github.com/cockroachdb/cockroach/rpc.(*Client).runHeartbeat(0xc820123c30, 0x3b9aca00, 0x6fc23ac00, 0x4000000000000000, 0x0, 0x0, 0xc8206a0780)
	/go/src/github.com/cockroachdb/cockroach/rpc/client.go:318 +0x807
github.com/cockroachdb/cockroach/rpc.NewClient.func1()
	/go/src/github.com/cockroachdb/cockroach/rpc/client.go:148 +0x36
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820545380)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25508 [select]:
github.com/cockroachdb/cockroach/rpc.(*RemoteClockMonitor).MonitorRemoteOffsets(0xc820a1a900, 0xc820476000)
	/go/src/github.com/cockroachdb/cockroach/rpc/clock_offset.go:156 +0x742
github.com/cockroachdb/cockroach/server.NewServer.func1()
	/go/src/github.com/cockroachdb/cockroach/server/server.go:123 +0x31
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820a1a920)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25695 [chan receive]:
database/sql.(*DB).connectionOpener(0xc82042a280)
	/usr/local/go/src/database/sql/sql.go:634 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:481 +0x336

goroutine 25714 [select]:
net/http.(*persistConn).writeLoop(0xc8201898c0)
	/usr/local/go/src/net/http/transport.go:1009 +0x40c
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:686 +0xc9d

goroutine 25514 [chan receive]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8202619a0)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x3a
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x490

goroutine 25526 [select]:
github.com/cockroachdb/cockroach/gossip.(*Gossip).manage.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/gossip.go:786 +0x330
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820a60540)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25510 [select]:
github.com/cockroachdb/cockroach/util.NewFeed.func1()
	/go/src/github.com/cockroachdb/cockroach/util/feed.go:61 +0x157
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc8204d18d0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25534 [IO wait]:
net.runtime_pollWait(0x7f0cadb54f18, 0x72, 0xc82007e0b0)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820206b50, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820206b50, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820206af0, 0xc820475000, 0x1000, 0x1000, 0x0, 0x7f0cadb44028, 0xc82007e0b0)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc820082488, 0xc820475000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/rpc.(*Client).input(0xc82065f980)
	/usr/local/go/src/net/rpc/client.go:109 +0xbf
created by net/rpc.NewClientWithCodec
	/usr/local/go/src/net/rpc/client.go:201 +0xd2

goroutine 25503 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x329
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820694c40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25523 [chan receive]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x5b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820a60480)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25522 [IO wait]:
net.runtime_pollWait(0x7f0cadb54e58, 0x72, 0xc82007e0b0)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820480d10, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820480d10, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).accept(0xc820480cb0, 0x0, 0x7f0cb039eb00, 0xc820228440)
	/usr/local/go/src/net/fd_unix.go:408 +0x27c
net.(*TCPListener).AcceptTCP(0xc820026480, 0xc800000000, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:254 +0x4d
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820a92ff0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25511 [select]:
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).startStats(0xc820103360)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:214 +0xca5
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).(github.com/cockroachdb/cockroach/kv.startStats)-fm()
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:201 +0x20
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc8204d1920)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25528 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1585 +0x8c7
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc8207ce5b0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25505 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x329
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820694c80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25524 [chan receive]:
github.com/cockroachdb/cockroach/gossip.(*server).start.func2()
	/go/src/github.com/cockroachdb/cockroach/gossip/server.go:243 +0x5e
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820a60500)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25538 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x329
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820694ca0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25549 [IO wait]:
net.runtime_pollWait(0x7f0cadb54a98, 0x72, 0xc82007e0b0)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc8203be920, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203be920, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).accept(0xc8203be8c0, 0x0, 0x7f0cb039eb00, 0xc820a61540)
	/usr/local/go/src/net/fd_unix.go:408 +0x27c
net.(*TCPListener).AcceptTCP(0xc820082450, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:254 +0x4d
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc8203ee040)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25545 [select]:
github.com/cockroachdb/cockroach/ts.(*poller).start.func1()
	/go/src/github.com/cockroachdb/cockroach/ts/db.go:76 +0x141
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820533b50)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25550 [chan receive]:
github.com/cockroachdb/cockroach/sql/pgwire.(*Server).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/sql/pgwire/server.go:73 +0x4a
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc8202c8250)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25509 [select]:
github.com/cockroachdb/cockroach/storage.(*StorePool).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store_pool.go:213 +0x1b6
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820a1aa80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25525 [select]:
github.com/cockroachdb/cockroach/gossip.(*Gossip).bootstrap.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/gossip.go:752 +0x2b4
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820a60520)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25507 [select]:
github.com/cockroachdb/cockroach/util/retry.(*Retry).Next(0xc82098fed0, 0x14d2b00)
	/go/src/github.com/cockroachdb/cockroach/util/retry/retry.go:106 +0x176
github.com/cockroachdb/cockroach/sql_test.TestAsyncSchemaChanger(0xc820a7c000)
	/go/src/github.com/cockroachdb/cockroach/sql/schema_changer_test.go:333 +0xbe0
testing.tRunner(0xc820a7c000, 0x1e03e88)
	/usr/local/go/src/testing/testing.go:456 +0x98
created by testing.RunTests
	/usr/local/go/src/testing/testing.go:561 +0x86d

goroutine 25697 [IO wait]:
net.runtime_pollWait(0x7f0cadb54798, 0x72, 0xc82007e0b0)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc8201b5b80, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8201b5b80, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8201b5b20, 0xc820644000, 0x1000, 0x1000, 0x0, 0x7f0cadb44028, 0xc82007e0b0)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc820026d78, 0xc820644000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8201898c0)
	/usr/local/go/src/net/http/transport.go:876 +0xf7
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:685 +0xc78

goroutine 25531 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:641 +0x3bf
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820a60cc0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25544 [select]:
github.com/cockroachdb/cockroach/server.(*Node).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/server/node.go:417 +0x15a
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820695b60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25547 [select]:
github.com/cockroachdb/cockroach/server.(*Server).startWriteSummaries.func1()
	/go/src/github.com/cockroachdb/cockroach/server/server.go:292 +0x169
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820533bb0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25703 [IO wait]:
net.runtime_pollWait(0x7f0cadb54cd8, 0x72, 0xc82007e0b0)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820692e60, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820692e60, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820692e00, 0xc820636000, 0x400, 0x400, 0x0, 0x7f0cadb44028, 0xc82007e0b0)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc820026d80, 0xc820636000, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*conn).serve(0xc820aa3e40)
	/usr/local/go/src/net/http/server.go:1319 +0x727
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:1910 +0x3f6

goroutine 25541 [select]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc820452000, 0xece26b6f2, 0x17b59b53, 0x214d960, 0xc820a02750, 0xc820476000, 0xc82052f040, 0xc8209afe00)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:176 +0x53f
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1.1(0xc82052f040, 0xc820a931d0)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:214 +0x78
github.com/cockroachdb/cockroach/storage.(*storeRangeSet).Visit(0xc820a93320, 0xc820775080)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:225 +0x246
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:216 +0x1e1
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820694d40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25554 [chan receive]:
github.com/cockroachdb/cockroach/rpc.(*Server).sendResponses(0xc8205b5140, 0x7f0cb03a39b0, 0xc8202d7b00, 0xc8206f2900)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:300 +0xa4
github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP.func1(0xc8205b5140, 0x7f0cb03a39b0, 0xc8202d7b00, 0xc8206f2900)
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:207 +0x3f
created by github.com/cockroachdb/cockroach/rpc.(*Server).ServeHTTP
	/go/src/github.com/cockroachdb/cockroach/rpc/server.go:208 +0x89c

goroutine 25548 [select]:
github.com/cockroachdb/cockroach/sql.(*SchemaChangeManager).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/schema_changer.go:507 +0x1732
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820695c80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25539 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x329
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820694ce0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25543 [select]:
github.com/cockroachdb/cockroach/server.(*Node).startPublishStatuses.func1()
	/go/src/github.com/cockroachdb/cockroach/server/node.go:445 +0x23e
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820695b40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25540 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x329
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820694d00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25513 [select]:
github.com/cockroachdb/cockroach/sql.NewExecutor.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/executor.go:97 +0x134
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820a03830)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25504 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:317 +0x329
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820694c60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25546 [select]:
github.com/cockroachdb/cockroach/ts.(*poller).start.func1()
	/go/src/github.com/cockroachdb/cockroach/ts/db.go:76 +0x141
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820533b70)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25529 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:562 +0x119
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc820476000, 0xc820a60c00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62
FAIL	github.com/cockroachdb/cockroach/sql	70.041s
=== RUN   TestNoLinkForbidden
--- PASS: TestNoLinkForbidden (0.11s)
=== RUN   TestDates
I0112 08:56:25.063336 842 rpc/clock_offset.go:153  monitoring cluster offset
I0112 08:56:25.065888 842 raft/raft.go:446  [group 1] 1 became follower at term 5
I0112 08:56:25.065997 842 raft/raft.go:234  [group 1] newRaft 1 [peers: [1], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I0112 08:56:25.066061 842 raft/raft.go:526  [group 1] 1 is starting a new election at term 5
I0112 08:56:25.066118 842 raft/raft.go:459  [group 1] 1 became candidate at term 6
I0112 08:56:25.066176 842 raft/raft.go:508  [group 1] 1 received vote from 1 at term 6
I0112 08:56:25.066234 842 raft/raft.go:487  [group 1] 1 became leader at term 6

```
Please assign, take a look and update the issue accordingly.

--- a/sql/schema_changer.go
+++ b/sql/schema_changer.go
@@ -158,9 +158,10 @@ func (sc *SchemaChanger) ReleaseLease(lease TableDescriptor_SchemaChangeLease) *
 }
 
 // ExtendLease for the current leaser.
-func (sc *SchemaChanger) ExtendLease(lease TableDescriptor_SchemaChangeLease) (TableDescriptor_SchemaChangeLease, *roachpb.Error) {
+func (sc *SchemaChanger) ExtendLease(existingLease TableDescriptor_SchemaChangeLease) (TableDescriptor_SchemaChangeLease, *roachpb.Error) {
+	var lease TableDescriptor_SchemaChangeLease
 	err := sc.db.Txn(func(txn *client.Txn) *roachpb.Error {
-		tableDesc, err := sc.findTableWithLease(txn, lease)
+		tableDesc, err := sc.findTableWithLease(txn, existingLease)
 		if err != nil {
 			return err
 		}
@@ -340,6 +341,7 @@ func (sc *SchemaChanger) purgeMutations(lease *TableDescriptor_SchemaChangeLease
 				// mutations if they have the mutation ID we're looking for.
 				break
 			}
+			log.Warningf("Purging schema change mutation: %v", desc.Mutations[i])
 			switch mutation.Direction {
 			case DescriptorMutation_ADD:
 				desc.Mutations[i].Direction = DescriptorMutation_DROP

--- a/sql/schema_changer_test.go
+++ b/sql/schema_changer_test.go
@@ -280,7 +280,6 @@ INSERT INTO t.test VALUES ('a', 'b'), ('c', 'd');
 
 func TestAsyncSchemaChanger(t *testing.T) {
 	defer leaktest.AfterTest(t)
-	t.Skip("#3764")
 	// Disable synchronous schema change execution so
 	// the asynchronous schema changer executes all
 	// schema changes.

